networks/testnet/genesis.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Previously it was assumed that the network configuration will be stored in subfolders within `deployment/persistent_chains`. Either move to this folder, or fix the docs (search for `persistent_chains` and fix in running-node.md, how-to.md, running-genesis-node.md)

Reply:
Moved